spec/rester/rspec_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rester/rspec'
+
+RSpec.describe Rester::DummyService, rester: 'spec/stubs/dummy_service_stub.yml' do
+  describe '/v1/tests' do
+    context 'GET' do
+      context 'With some context' do
+        it { is_expected.to eq stub_response }
+      end
+
```

Comment:
We should test that it actually producing the subject/stub_response that we're expecting.

For example, if it was setting both the subject and stub_response to `nil`, then these tests would pass, and we wouldn't see that it was broken.